flink-core/src/main/java/org/apache/flink/core/fs/local/LocalDataOutputStream.java

```diff
@@ -35,6 +38,8 @@
     /** The file output stream used to write data. */
     private final FileOutputStream fos;
 
+    private volatile boolean isClosed = false;
```

Comment:
`isClosed` -> `closed`; the prefix `is` prefix is for boolean getters

```suggestion
    private boolean closed = false;
```